pootle/checks.py

```diff
@@ -340,6 +340,81 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W019",
                 ))
 
+    try:
+        edit_coef = settings.POOTLE_SCORING_CONF['EDIT_COEF']
+        try:
+            edit_coef + 0.0
+        except TypeError:
+            errors.append(checks.Critical(
+                _("Invalid value for EDIT_COEF in POOTLE_SCORING_CONF."),
+                hint=_("Set a valid value for EDIT_COEF "
+                       "in POOTLE_SCORING_CONF."),
```

Comment:
I'm checking that it has a numeric value. From common sense it should be less than 1, and sum of `edit_coef` and `review_coef` should be `1` . But I don't think it's worth to check these conditions here.